docs/content/glossary/glossary.mdx

```diff
@@ -0,0 +1,318 @@
+---
+title: Glossary
+---
+
+# Flow Glossary
+
+## All words and definitions related to Blockchain and Flow
+
+## **A**
+
+## Authentication
+[Authentication](https://docs.onflow.org/fcl/reference/authentication/#gatsby-focus-wrapper) connects users to their wallets, 
+and the wallet provides the account information and services through FCL.
+
+## Authorization
+Authorization is a process by which a server determines if the client has permission to use a resource or access a file. In FCL, 
+An authorization service knows the Account and the Key to sign the transaction when the service is sent to FCL during authentication.
+
+## Authorization Envelope
+The transaction authorization envelope contains both the [transaction payload](https://docs.onflow.org/concepts/transaction-signing/#payload) and the payload signatures.
+
+## Authorizers
+[Authorizers](https://docs.onflow.org/flow-go-sdk/#transactions) are accounts that authorize a transaction to read and mutate their 
+resources. A transaction can specify zero or more authorizers, depending on how many accounts the transaction needs to access.
+
+## Account
+An [account](https://docs.onflow.org/flow-token/concepts/#segmented-transaction-fees) on Flow is a record in the chain state that holds 
```

Comment:
code as in Cadence contracts?

Reply:
yes.
Cadence contracts deployed to the account